apig_wsgi.py

```diff
@@ -67,11 +68,10 @@ def get_environ(event, binary_support):
 
         environ['HTTP_' + key] = value
 
+    # Sends any extra context passed by a custom lambda authorizer
     request_context = event.get("requestContext") or {}
-    authorizer = request_context.get("authorizer") or {}
-    for key, value in authorizer.items():
-        key = key.upper().replace("-", "_")
-        environ["HTTP_X_AUTHORIZER_" + key] = value
+    if "authorizer" in request_context:
+        environ["AUTHORIZER"] = json.dumps(request_context["authorizer"])
```

Comment:
Why add the JSON dumping? This doesn't seem necessary, as it makes consumers undump it?

Reply:
After consideration I think the cleanest solution would be to provide the full context, if it's provided in the event, at the key `api-gateway.request_context`

This way if/when AWS expand this 'context' further, applications will be able to consume it with no work in the library, and the name will avoid potential conflicts with other WSGI variables.

Reply:
Thanks! I will do those changes.

I was playing safe with the JSON dumps because an environment variable can only be a string, but I think that it makes sense to just pass the object avoid the extra runtime cost.

Reply:
WSGI `environ` is a separate concept to `os.environ` :)